pkg/scheduler/framework/plugins/volumebinding/volume_binding.go

```diff
@@ -89,6 +90,23 @@ func (pl *VolumeBinding) Name() string {
 	return Name
 }
 
+// EventsToRegister returns the possible events that may make a Pod
+// failed by this plugin schedulable.
+func (pl *VolumeBinding) EventsToRegister() []framework.ClusterEvent {
+	return []framework.ClusterEvent{
+		// Pods may fail because the storage class is not configured correctly.
+		// After the storage class is fixed, the pods may become schedulable.
```

Comment:
do we need {StorageClass, Update}?

Reply:
https://github.com/kubernetes/kubernetes/blob/4740173f3378ef9d0dc59b0aa9299444a97d0818/pkg/apis/storage/validation/validation.go#L68-L85
checked again, although most fields are immutable, but `AllowedTopologies` is not, so the `Update` event is required.